Source/stores.cpp

```diff
@@ -1,3 +1,8 @@
+/**
+ * @file stores.cpp
+ *
+ * Implementation of functionality for stores and yoenrt fislohd.

```

Comment:
what is `yoenrt fislohd`?

Reply:
"towner dialogs", but with left hand +1 in qwerty